tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP hep-ex</categories>
+          <categories>math-ph math.MP</categories>
```

Comment:
I had to remove this category otherwise this record would have been skipped because the category is already harvested on legacy.